StripeUICore/StripeUICore/Source/Validators/STPBlikCodeValidator.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  STPBlikCodeValidator.swift
+//  StripeUICoreTests
+//
+//  Created by Fionn Barrett on 07/07/2023.
+//
+
+import Foundation
+
+@_spi(STP) public class STPBlikCodeValidator: NSObject {
```

Comment:
Does this need to inherit from NSObject? I see some of our other validators do but it's not needed AFAIK.